src/UI/class-settings-page.php

```diff
@@ -1125,20 +1125,12 @@ private function validate_options_post_type_tracking( array &$input ): void {
 			}
 
 			// Cleanup and sanitized values assignment.
-			unset( $input[ $track_as ] );
 			$input [ $posts ] = self::sanitize_option_array( $temp_posts );
 			$input [ $pages ] = self::sanitize_option_array( $temp_pages );
-		} else {
-			add_settings_error(
-				Parsely::OPTIONS_KEY,
-				$track_as,
-				sprintf(
-					__(
-						'Settings could not be saved because post tracking data is invalid.',
-						'wp-parsely'
-					)
-				)
-			);
```

Comment:
Is this error message no longer relevant?

Reply:
Correct. We don't display an error message anymore. If the given data is empty (which is the case when the recrawl settings section is invisible) or tampered/invalid in any way, the data in the database remains as it is, preserving data integrity.